src/workers/jobs.js

```diff
@@ -295,8 +295,12 @@ export async function completeContactLoad(
     .knex("campaign_admin")
     .where("campaign_id", campaignId)
     .update({
-      deleted_optouts_count: deleteOptOutCells || null,
-      duplicate_contacts_count: deleteDuplicateCells || null,
+      deleted_optouts_count:
+        deleteOptOutCells || deleteOptOutCells === 0 ? deleteOptOutCells : null,
```

Comment:
maybe we should just set them to `null` higher up where they're declared in `let deleteOptOutCells`  and then they can be fully set here without `||`'s, etc?

Reply:
Done!